src/app/components/media/MediaComponentRightPanel.js

```diff
@@ -89,6 +104,7 @@ const MediaComponentRightPanel = ({
       { showTab === 'suggestedMedia' ? <MediaSuggestions dbid={projectMedia.dbid} teamDbid={projectMedia.team?.dbid} superAdminMask={superAdminMask} /> : null }
       { showTab === 'metadata' ? <MediaTasks media={projectMedia} fieldset="metadata" /> : null }
       { showTab === 'source' ? <MediaSource projectMedia={projectMedia} /> : null }
+      { showTab === 'articles' ? <ArticlesSidebar teamSlug={projectMedia.team.slug} projectMediaDbid={projectMedia.dbid} /> : null }
```

Comment:
Maybe call this component `MediaArticles` for consistency?

Reply:
Sure, I'll rename!

Reply:
This is beauty in component naming form! :star_struck: